pages/status.tsx

```diff
@@ -148,6 +154,7 @@ const Status: FC = () => {
       <h1 className="h1">{t('header')}</h1>
       {(() => {
         if (checkStatusResponse !== undefined) {
+          scrollToTop()
```

Comment:
This will make the page scroll everytime the component re-render. Not recommended. If you want to scroll to top on event then you can add event handler on `useCheckStatus` options.